src/main/java/application/rest/v1/configmaps/KindActionMappingProcessor.java

```diff
@@ -0,0 +1,462 @@
+        return configMapsList;
+    }
+
+    /**
+     * Get all "KindActionMapping" custom resources in a cluster
+     * 
+     * @param client
+     * @return
+     * @throws ApiException
+     */
+    protected List<JsonObject> listKAMCustomResources(ApiClient client) 
+        throws ApiException {
+        final CustomObjectsApi coa = new CustomObjectsApi();
+        coa.setApiClient(client);
+        if (Logger.isDebugEnabled()) {
+            Logger.log(className, "listKAMCustomResources", Logger.LogType.DEBUG, 
+                "\n List KAM Custom Resources for all namespaces with" +
+                "\n group = " + "actions.kappnav.io" + 
+                "\n namespace = kappnav and name = default");
+        }
+
+        Object kamResource = coa.listClusterCustomObject(KAM_GROUP, KAM_VERSION, KAM_PLURAL, null, 
+                             null, null, null);
+        return KAppNavEndpoint.getItemAsList(client, kamResource);
+    }
+
+    /**
+     * Get the namespace for the given KindActionMapping resource
+     * 
+     * @param kam
+     * @return the namespace for the given kam resource
+     */
+    private String getKAMNamespace(JsonElement kam) {
+        if (Logger.isEntryEnabled())
+            Logger.log(className, "getKAMNamespace", Logger.LogType.ENTRY,"");
+
+        String namespaceStr = null;
+        JsonElement metadata = (JsonObject) kam.getAsJsonObject().get(METADATA_PROPERTY_NAME);
+        if (metadata != null) {
+            JsonObject namespace= metadata.getAsJsonObject();
+            if (namespace != null) 
+                namespaceStr = namespace.get(NAMESPACE_PROPERTY_NAME).getAsString();
+        } else {
+            if (Logger.isDebugEnabled()) 
+                Logger.log(className, "getKAMNamespace", Logger.LogType.DEBUG, "kam metadata is null.");
+        }
+
+        if (Logger.isExitEnabled()) {
+            Logger.log(className, "getKAMNamespace", Logger.LogType.EXIT, "kam namespace = " + namespaceStr);
+        } 
+        return namespaceStr;
+    }
+
+    /**
+     * Check to see if the apiVersions of a component and a mapping are matching
+     * 
+     * @param apiVersion
+     * @param compApiVersion
+     * @return
+     */
+    private boolean isApiVersionMatch(String apiVersion, String compApiVersion) {
+        if (Logger.isEntryEnabled()) 
+            Logger.log(className, "isApiVersionMatch", Logger.LogType.ENTRY, "apiVerion = " + apiVersion +
+                       ", compApiVersion = " + compApiVersion);
+
+        boolean match = false;
+        if ( (apiVersion != null) && (!apiVersion.isEmpty()) &&
+             (compApiVersion != null) && (!compApiVersion.isEmpty()) ) {   
+            if (apiVersion.equals(compApiVersion)) { // group/version - exact match
+                match = true;
+            } else {
+                String grp_version[] = apiVersion.split("/");
+                String comp_grp_version[] = compApiVersion.split("/");
```

Comment:
Might be good to add a DEBUG log entry to display the split results.  If grp_version and comp_grp_version array lengths differ, this function ends silently with match=false